src/libslic3r/GCodeWriter.cpp

```diff
@@ -325,6 +343,32 @@ std::string GCodeWriter::write_acceleration(){
     return gcode.str();
 }
 
+std::string GCodeWriter::write_pressure(){// adding it in now for future commits of pressure advance numbers.
```

Comment:
don't add things that aren't useful. Just store it into another commit, and submit them when it's implemented.

Reply:
yeah that's fair. just wanted it there as a "sneak peek"